core/lib/libmbedtls/src/sub.mk

```diff
@@ -1,2 +1,16 @@
 srcs-y += tee_lmd_provider.c
-#cflags-tee_lmd_provider.c-y += -Wno-unused-parameter
+
+# For mbedtls
+cflags-y += -Icore/lib/libmbedtls/src/mbedtls/include
+cflags-y += -D'MBEDTLS_CONFIG_FILE="mbedtls_config.h"'
+cflags-y += -D_FILE_OFFSET_BITS=64 -Wno-old-style-definition -Wno-redundant-decls
+
+# Hashes
+srcs-$(CFG_CRYPTO_MD5) += mbedtls/library/md5.c
```

Comment:
I'd rather compile these files unconditionally, the mbedTLS config file is supposed to take care of excluding what's not configured.

Reply:
This comment is not answered

Reply:
The patch is ongoing, I will update to fix it.